forge-gui/res/cardsfolder/upcoming/into_the_flood_maw.txt

```diff
@@ -0,0 +1,10 @@
+Name:Into the Flood Maw
+ManaCost:U
+Types:Instant
+K:Gift
+SVar:GiftAbility:DB$ Token | TokenAmount$ 1 | TokenScript$ u_1_1_fish | TokenTapped$ True | TokenOwner$ Promised | GiftDescription$ a tapped Fish

```

Comment:
the predefined creature tokens need LockTokenScript

Reply:
Actually, I wonder what would happen if you changed "Fish" to "Octopus" on a gift card :)

Reply:
well I'm not sure it's for this reason but they don't want it to work :p